### PR TITLE
Initial setup

### LINKED ISSUE #7: browser bundling tooling
We need to choose  if we will use a bundler at all, and if so, whic one (Parcel, Browserify, Webpack...), and set it up.

### LINKED ISSUE #15: Setup eslint + prettier

### LINKED ISSUE #16: Setup commitzen
I'll use the `cz-conventional-changelog` initially but we can change that if you guys prefer to use the `cz-lerna-changelog` even though we wont have any workspaces here or even another adapter

### PR DESCRIPTION
**Pull request by alcferreira**
- closes #7 
- closes #15
- closes #12 
- closes #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,5 +19,31 @@
...
   },
   "homepage": "https://github.com/CraveFood/webmap#readme",
-  "keywords": ["map", "slideshoow", "tv", "digital signaling"]
+  "keywords": ["map", "slideshoow", "tv", "digital signaling"],
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "devDependencies": {
```
Comment (vNakamura) [inline package.json]: Please add `parcel-bundler`.

Comment (alcferreira) [reply package.json]: opss

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
```
Comment (fczuardi) [inline .gitignore]: will we have this folder?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+# Optional npm cache directory
+.npm
+
+# Yarn Integrity file
+.yarn-integrity
+
+.cache/
+dist/
```
Comment (fczuardi) [inline .gitignore]: will we have this folder?

File: AUTHORS [hunk 4]
```diff
@@ -2,4 +2,7 @@
 
   Crave Food Systems <https://github.com/CraveFood> (http://cravefoodsystems.com/)
 
+  André L. C. Ferreira <alcf.2099@gmail.com> <alcferreira>
   Fabricio C Zuardi <fabricio@fabricio.org> (http://fabricio.org)
+  Felipe Gameiro <fgameiro>
+  Vinicius Nakamura <vNakamura>
```
Comment (fczuardi) [inline AUTHORS]: Lets use the short format used by npm https://docs.npmjs.com/files/package.json#people-fields-author-contributors

"Name <email> (url)"

**Comment by alcferreira** [comment]
Parcel didn't work for us... Will go with the create-react-app

**Pull request closed without merging**